docs/src/test/java/docs/http/javadsl/HttpServerActorInteractionExample.java

```diff
@@ -121,26 +115,20 @@ static Props props() {
       return Props.create(Auction.class);
     }
 
-    public Auction() {
-      receive(ReceiveBuilder.
-        match(HttpServerActorInteractionExample.Bid.class, bid -> {
-          bids.add(bid);
-          log.info("Bid complete: {}, {}", bid.userId, bid.offer);
-        }).
-        match(HttpServerActorInteractionExample.GetBids.class, m -> {
+    @Override
+    public Receive createReceive() {
+      return receiveBuilder()
+        .match(
+          HttpServerActorInteractionExample.Bid.class, bid -> {
+            bids.add(bid);
+            log.info("Bid complete: {}, {}", bid.userId, bid.offer);
+            })
```

Comment:
Maybe adjust indentation here as well (though this whole file has non-standard Java indentation anyway).